### ISSUE TITLE
Direct Transfer results

### ISSUE THREAD

**Issue by liangbh6** [Simon4Yan/Learning-via-Translation#1]
Hi, I have some questions about your 'Direct Transfer' results in Table 2 in your paper. I make the setting consistent with yours, but I can't get so-high baselines, like rank1 33.1%, mAP 16.7  when train on market1501 and test on Duke-MTMC-reID. Actually I always got this result around rank1 27%, mAP 13%. And this result is better than those in the [https://arxiv.org/pdf/1705.10444.pdf](url), where this result is that rank1 21.9%, mAP 10.9%. 
     I wonder that is there some tricks used in your experiments, and I'm looking forward to your reply .

**Comment by Simon4john** [comment]
@liangbh6 @Simon4Yan Which framework you use? Caffe or pytorch?

**Comment by liangbh6** [comment]
@Simon4john pytorch. So, the reason is the differences between pytorch and caffe? If I want to reproduce your results using pytorch, do you have some suggestion, about the learning rate, data augmentation, or testing tricks like normalization? Actually I have tried to normalized the features but it helped a little.

**Comment by Simon4john** [comment]
I think it is the difference between frameworks. Maybe we can discuss this problem by wechat, that will be more convenient. My e-mail is 12213033@bjtu.edu.cn.

**Comment by Simon4Yan** [comment]
@liangbh6 @Simon4john Thanks for your attention. The code for re-ID feature learning is mainly modified from [IDE](https://github.com/zhunzhong07/IDE-baseline-Market-1501), and the framework is Caffe.

**Comment by Simon4Yan** [comment]
Thanks for your question. And we conduct experiment to see that difference between pytroch and caffe, we find the BN leads to the this performance gap. I will give the experiment details about it after I come back to school.

**Comment by liangbh6** [comment]
@Simon4Yan Excellent work! Thanks a lot.
